pytorch/Dockerfile

```diff
@@ -253,13 +252,27 @@ else \n\
 fi \n\
 tail -f /dev/null" >> /usr/local/bin/dockerd-entrypoint.sh
 
-FROM torchserve-base AS torchserve
+FROM torchserve-base AS compile-cpu
+
+COPY serving/torchserve-requirements.txt .
+COPY requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r requirements.txt && \
+    python -m pip install --no-cache-dir -r torchserve-requirements.txt
+
+FROM torchserve-base AS compile-xpu
+
+COPY serving/torchserve-xpu-requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r torchserve-xpu-requirements.txt
```

Comment:
can we make sure to delete the requirements.txt files after they are installed?

Reply:
deleted